client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/thirdpartyclasses/HashOperationsClassReplacement.java

```diff
@@ -0,0 +1,77 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement.thirdpartyclasses;
+
+import org.evomaster.client.java.instrumentation.RedisCommand;
+import org.evomaster.client.java.instrumentation.RedisKeySchema;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.Replacement;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.ThirdPartyMethodReplacementClass;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.UsageFilter;
+import org.evomaster.client.java.instrumentation.object.ClassToSchema;
+import org.evomaster.client.java.instrumentation.shared.ReplacementCategory;
+import org.evomaster.client.java.instrumentation.shared.ReplacementType;
+import org.evomaster.client.java.instrumentation.staticstate.ExecutionTracer;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Collections;
+
+/**
+ * This replacement captures operations when Redis is integrated using Lettuce or Redisson.
+ * For other integrations using CrudRepositories, RedisKeyValueTemplateClassReplacement was included.
+ */
+public class HashOperationsClassReplacement extends ThirdPartyMethodReplacementClass {
+
+    private static final HashOperationsClassReplacement singleton = new HashOperationsClassReplacement();
+
+    @Override
+    protected String getNameOfThirdPartyTargetClass() {
+        return "org.springframework.data.redis.core.HashOperations";
+    }
```

Comment:
is it possible to perform the class replacements on the underlying Redis library classes, instead of the spring-data classes?